doc/content/guides/getting-started/configuration.md

```diff
@@ -44,103 +44,104 @@ Before we go into the details of The Things Stack's configuration, we'll take a
 
 The `ports` section exposes The Things Stack's ports to the world. Port `80` and `443` are mapped to the internal HTTP and HTTPS ports. The other ports have a direct mapping. If you don't need support for gateways and applications that don't support TLS, you can remove ports starting with `188`.
 
-```yml
+```yaml
 stack:
-  image: thethingsnetwork/lorawan-stack:<the tag>
-  entrypoint: ttn-lw-stack
-  command: start
-  restart: unless-stopped
+  image: 'thethingsnetwork/lorawan-stack:<the tag>'
+  entrypoint: 'ttn-lw-stack'
+  command: 'start'
+  restart: 'unless-stopped'
   depends_on:
-    - cockroach
-    - redis
+    - 'cockroach'
+    - 'redis'
   volumes:
-    - ./acme:/var/lib/acme
-    - ./data/blob:/srv/ttn-lorawan/public/blob
+    - './acme:/var/lib/acme'
+    - './data/blob:/srv/ttn-lorawan/public/blob'
   ports:
-    - "80:1885"
-    - "443:8885"
-    - "1882:1882"
-    - "8882:8882"
-    - "1883:1883"
-    - "8883:8883"
-    - "1884:1884"
-    - "8884:8884"
-    - "1887:1887"
-    - "8887:8887"
-    - "1700:1700/udp"
-```
-
-Next, we'll have a look at the configuration options for your private deployment. These options are set in the `environment` section of the `stack` service in `docker-compose.yml`.
+    - '80:1885'
+    - '443:8885'
+    - '1882:1882'
+    - '8882:8882'
+    - '1883:1883'
+    - '8883:8883'
+    - '1884:1884'
+    - '8884:8884'
+    - '1887:1887'
+    - '8887:8887'
+    - '1700:1700/udp'
+  env_file: '.env'
+```
+
+Next, we'll have a look at the configuration options for your private deployment. We'll set these options in the `.env` file that is referenced by the `env_file` option of the `stack` service in `docker-compose.yml`.
 
 First we'll make sure that The Things Stack uses the correct databases.
 
-```
-TTN_LW_IS_DATABASE_URI=postgres://root@cockroach:26257/ttn_lorawan?sslmode=disable
-TTN_LW_REDIS_ADDRESS=redis:6379
+```bash
```

Comment:
This is still in yaml right? Or do you mean this is `env` stuff that can be injected in many ways?
Same with the other occurences

Reply:
It can be injected in multiple ways indeed.